src/DevChatter.Bot.Core/Games/DealNoDeal/DealNoDealGame.cs

```diff
@@ -0,0 +1,342 @@
+using DevChatter.Bot.Core.Automation;
+using DevChatter.Bot.Core.Data.Model;
+using DevChatter.Bot.Core.Events;
+using DevChatter.Bot.Core.Extensions;
+using DevChatter.Bot.Core.Systems.Chat;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DevChatter.Bot.Core.Games.DealNoDeal
+{
+    public class DealNoDealGame : IGame
+    {
+        public bool IsRunning { get; private set; }
+
+        //Configuration
+        public const UserRole ROLE_REQUIRED_TO_START = UserRole.Everyone;
+        public const int SECONDS_TO_CHOOSE_BOXES = 30;
+        public const int TOKENS_TO_PLAY = 25;
+        public const int MIN_PLAYABLE_BOXES = 5;
+        //Change these if you don't like the rewards
+        private static readonly List<int> InitialPrices = new List<int>()
+        {
+            //15 at the moment
+            0,1,2,4,6,8,10,15,18,20,25,32,50,64,100
+        };
+
+        public List<Box> StartingBoxes;
+        public List<Box> BoxesWithOwners;
```

Comment:
Would it make sense to initialize these to empty list to avoid `NullReferenceException`?